<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-01T15:36:30 IST" name="Default suite" finished-at="2023-01-01T15:36:35 IST" duration-ms="4799">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.TestRunnerClass@48f278eb]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-01-01T15:36:30 IST" name="Default test" finished-at="2023-01-01T15:36:35 IST" duration-ms="4799">
      <class name="testRunner.TestRunnerClass">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:testRunner.TestRunnerClass@48f278eb]" started-at="2023-01-01T15:36:30 IST" name="setUpClass" finished-at="2023-01-01T15:36:31 IST" duration-ms="418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.TestRunnerClass@48f278eb]" started-at="2023-01-01T15:36:31 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-01-01T15:36:35 IST" duration-ms="4296" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["open browser and  enter the test url login page should be displayed,Enter valid Username and valid password"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at StepDefination.steps.home_page_should_be_displayed(steps.java:54)
at âœ½.Home page should be displayed(file:///C:/Users/Public/com.Tyspltd_cucumber/Featurefile/Login.feature:11)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:testRunner.TestRunnerClass@48f278eb]" started-at="2023-01-01T15:36:35 IST" name="tearDownClass" finished-at="2023-01-01T15:36:35 IST" duration-ms="33" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunner.TestRunnerClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
